torch/onnx/__init__.py

```diff
@@ -188,18 +188,18 @@ def export(model, args, f, export_params=True, verbose=False, training=TrainingM
                   %3 : Float = onnx::Mul(%2, %0)
                   return (%3)
 
-              If an op is in the TorchScript namespace "quantized", it will be exported
-              in the ONNX opset domain "caffe2". These ops are produced by
-              the modules described in
+              If PyTorch was built with Caffe2 (i.e. with ``BUILD_CAFFE2=1``), then
+              Caffe2-specific behavior will be enabled, including special support
+              for ops are produced by the modules described in
               `Quantization <https://pytorch.org/docs/stable/quantization.html>`_.
 
               .. warning::
 
                 Models exported this way are probably runnable only by Caffe2.
 
-        opset_version (int, default 9):
-            Must be ``== _onnx_main_opset or in _onnx_stable_opsets``,
-            defined in torch/onnx/symbolic_helper.py.
+        opset_version (int, default 9): The version of the
+            `default opset <https://github.com/onnx/onnx/blob/master/docs/Operators.md>`_
```

Comment:
what does `default opset` mean?

Reply:
If an op doesn't specify a domain, it's in the default opset AKA `ai.onnx`.
I guess if it's not clear to you then it definitely won't be clear to many people.
Any suggestions as to how to write this to make it clearer?

Reply:
exchanging `default` with `ai.onnx` or `ONNX` or `default (ai.onnx)`?